tests/src/Functional/DatasetBTBTest.php

```diff
@@ -129,6 +128,68 @@ public function testHarvestOrphan() {
     $this->assertEquals('published', $this->getModerationState('4'));
   }
 
+  /**
+   * Test cleanup of orphaned draft distributions.
+   */
+  public function testOrphanDraftDistributionCleanup() {
+    // Get the original configuration settings.
+    $config = \Drupal::service('config.factory');
+    $datastoreSettings = $config->getEditable('datastore.settings');
+    $deleteLocalResourceOriginal = $datastoreSettings->get('delete_local_resource');
+    $triggeringOriginal = $datastoreSettings->get('triggering_properties');
+    $defaultModerationState = $config->getEditable('workflows.workflow.dkan_publishing');
+    $defaultModerationStateOriginal = $defaultModerationState->get('type_settings.default_moderation_state');
```

Comment:
Since this is a BTB test, you don't need to save the original state and then restore it later.